src/calculator/routes.ts

```diff
@@ -19,6 +20,15 @@ calculatorRoutes.get('/twitter/user/:id', function(req, res, next) {
     })
 })
 
+calculatorRoutes.get('/scrapper/user-verified/:verified/user-joined/:joined', function(req, res, next) {
+  const userCredibility = scrapperUserCredibility(req.params.verified == 'true', Number(req.params.joined))
```

Comment:
Please, compare with `===` instead of `==`. `==` in js is weaker than `===`. Things like this:
```javascript
1 == "1" // true
1 === "1" // false
```
could happen.

Reply:
done